src/cmd/create.rs

```diff
@@ -12,13 +12,22 @@ use std::{
     path::{Path, PathBuf},
 };
 
+use strum_macros::EnumString;
+
 #[derive(Debug, StructOpt)]
 /// Create a new wallet
 pub enum Cmd {
     Basic(Basic),
     Sharded(Sharded),
 }
 
+#[derive(Copy, Clone, Debug, PartialEq, EnumString)]
+#[strum(serialize_all = "mixed_case")]
+pub enum SeedType {
```

Comment:
Move this definition to the mnemonic level? importing "cmd" types into non cmd code is not a good convention and that seems to be where it's actually used

Reply:
done.